examples/platform/silabs/efr32/rs911x/rsi_wlan_config.h

```diff
@@ -53,20 +53,29 @@
 #endif
 
 //! To set Extended custom feature select bit map
-#if WIFI_ENABLE_SECURITY_WPA3
+#if WIFI_ENABLE_SECURITY_MODE_WPA3_TRANSITION /* TRANSITION MODE:WPA2 AND WPA3 SUPPORTED */
```

Comment:
It is not required since it is already available in the WPA3 only mode. The configurations seems to be the same for WPA3 only and WPA3 transition mode